README.md

```diff
@@ -213,3 +213,6 @@ This tutorial uses the MedNIST scan (or alternatively the MNIST) dataset to demo
 
 #### [interpretability](./modules/interpretability)
 Tutorials in this folder demonstrate model visualisation and interpretability features of MONAI. Currently, it consists of class activation mapping and occlusion sensitivity for 3D classification model visualisations and analysis.
+
+#### [Transfer learning with MMAR](./modules/transfer_mmar.ipynb)
+This tutorial demonstrates a transfer learning pipeline from a pretrained model in [Clara Train's Medical Model Archive format](https://docs.nvidia.com/clara/clara-train-sdk/pt/mmar.html).
```

Comment:
Could you please help add a description here that it also demo how to use `LMDBDataset`?